hashicorp/vault
# [secrets sync] hide sync content from client overview

### :hammer_and_wrench: Description
Hide secrets sync content from clients overview page if:
- cluster is from HVD
- license doesn't include it
- feature is not activated

### 🔗 Links
follow up to https://github.com/hashicorp/vault/pull/26024

### :camera_flash: Screenshots
### OSS
![image](https://github.com/hashicorp/vault/assets/903288/743f41db-16fb-4d1c-9d74-6f9029b1f8a5)

### Enterprise + feature not activated
![Screenshot 2024-03-21 at 5 15 57 PM](https://github.com/hashicorp/vault/assets/903288/a09a7e7e-629b-42ba-a451-61e258db0661)
![Screenshot 2024-03-21 at 5 16 05 PM](https://github.com/hashicorp/vault/assets/903288/7e0971ed-0690-48e4-b225-f21f3cd0c584)

### Enterprise + feature activated, but no clients
![Screenshot 2024-03-21 at 5 42 13 PM](https://github.com/hashicorp/vault/assets/903288/4c1960b9-d6c8-4b42-8153-62160cb04ed4)
![Screenshot 2024-03-21 at 5 42 07 PM](https://github.com/hashicorp/vault/assets/903288/c9f79c97-8b9d-4771-a574-9d9175bed365)


### :building_construction: How to Build and Test the Change
With an OSS license:
1. Visit `/vault/clients/counts/overview`
2. You should NOT see any secrets sync related content

With an ENT license that doesn't have secrets sync enabled:
_same steps as OSS above_

With an ENT license that has secrets sync enabled:
1. Visit `/vault/clients/counts/overview`
2. You should see the secrets sync tab and chart
